### ISSUE TITLE
SDWebImagePrefetcher no longer works

### ISSUE THREAD

**Issue by houmie** [rs/SDWebImage#1426]
```
    [[SDWebImagePrefetcher sharedImagePrefetcher] prefetchURLs:urls];
```

This used to work fine in pod 'SDWebImage', '3.7.3'.  

2016-01-12 19:03:36.181 itagnews[84199:4788062] Prefetched 1 out of 68
2016-01-12 19:03:36.200 itagnews[84199:4788062] Prefetched 2 out of 68
2016-01-12 19:03:36.240 itagnews[84199:4788062] Prefetched 3 out of 68
2016-01-12 19:03:36.339 itagnews[84199:4788062] Prefetched 4 out of 68
2016-01-12 19:03:36.359 itagnews[84199:4788062] Prefetched 5 out of 68
2016-01-12 19:03:36.461 itagnews[84199:4788062] Prefetched 6 out of 68
2016-01-12 19:03:36.480 itagnews[84199:4788062] Prefetched 7 out of 68

But in pod 'SDWebImage', '3.7.4', this no longer works. I don't even get to see it in the console, nor are the images prefetched. They get loaded after I scroll down.  I just had to downgrade the version to the previous version.

Thanks,
Houman

**Comment by kaandedeoglu** [comment]
Same here.

**Comment by bpoplauschi** [comment]
Fixed by a8fca65
